src/js/view/controls/components/menu/settings-menu.js

```diff
@@ -138,13 +138,13 @@ const SettingsMenu = (api, model, controlbar, localization) => {
                 newStyles[propertyName] = newValue;
                 model.set('captions', newStyles);
             };
-            const resetItem = new RadioMenuItem('Reset', () => {
-                model.set('captions', CaptionsDefaults);
-                renderCaptionsSettings();
-            });
-            resetItem.el.classList.add('jw-settings-reset');
             const persistedOptions = model.get('captions');
             const renderCaptionsSettings = () => {
+                const resetItem = new MenuItem('Reset', () => {
```

Comment:
can you explain the difference between `RadioMenuItem` and `MenuItem` and how that helps with this exception?

also are we sure that on render, the previous `resetItem` is garbage collected and we aren't creating a memory leak here?

Reply:
Yep, it is because it's added when we call setItem, so it's destroyed whenever it should be. RadioMenuItems will also deactivate any active menu items on activation, as well as add an attribute. This isn't a radio button option, so these additional methods were unnecessary and did nothing.